metron-deployment/roles/kibana/README.md

```diff
@@ -1,35 +0,0 @@
-Kibana 4
```

Comment:
The file I commented was the readme that documented how to add or modify the kibana page template